README.md

```diff
@@ -93,11 +93,16 @@ interpreter in `$PATH` for the Python version of the code you're analyzing.
 
 Platform support:
 
-* Pytype is currently developed and tested on Linux, which is the main supported
+* Pytype is currently developed and tested on Linux\*, which is the main supported
   platform.
 * Installation on MacOSX requires OSX 10.7 or higher and Xcode v8 or higher.
 * Windows is currently not supported unless you use [WSL][wsl].
 
+<sub>\*
+    Note: Pytype is tested on Ubuntu Linux.
+    On Alpine Linux, installing may fail due to issues with upstream dependencies.
```

Comment:
Could you add a link to one of the upstream issues? Maybe https://github.com/scikit-build/ninja-python-distributions/issues/27, since there's a possible workaround suggested there (adding the missing scikit-build dependency in a pyproject.toml file).

Reply:
Added in 91a62302.